cpp/src/arrow/array.h

```diff
@@ -512,6 +512,7 @@ class ARROW_EXPORT FixedSizeBinaryArray : public PrimitiveArray {
                        int64_t null_count = 0, int64_t offset = 0);
 
   const uint8_t* GetValue(int64_t i) const;
```

Comment:
Should we deprecate this?

Reply:
I would keep it for now to be consistent with `BinaryArray`.  At the moment it makes sense for consistency to have both methods: `Value()` for consistency with fixed width types and `GetValue()` for consistency with binary types.